jetcd-core/src/main/java/io/etcd/jetcd/cluster/Member.java

```diff
@@ -38,6 +39,15 @@ public long getId() {
         return member.getID();
     }
 
+    /**
+     * Returns the member ID for this member.
+     *
+     * @return the unsigned id.
+     */
+    public BigInteger getUnsignedId() {
+        return new BigInteger(Long.toUnsignedString(getId()));
```

Comment:
Is there a way to avoid the intermediate string representation ? I'd like to avoid generating garbage if possible.